### DISCUSSION TITLE
Filament upload in Relation managers

### DISCUSSION THREAD

**Discussion by benbelgacemhichem** [filamentphp/filament#10494] [Help]
The package always return error:  
General error: 1364 Field 'collection_name' doesn't have a default value

SpatieMediaLibraryFileUpload::make('media')
                    ->multiple()
                    ->reorderable()

**Comment by leandrocfe** [comment]
Did you config the withPivot? Could you provide more info please?

**Reply by benbelgacemhichem** [reply]
SpatieMediaLibraryFileUpload Not working with Relation manager

SQLSTATE[HY000]: General error: 1364 Field 'collection_name' doesn't have a default value
INSERT INTO
  `media` (
    `model_id`,
    `model_type`,
    `uuid`,
    `order_column`,
    `updated_at`,
    `created_at`
  )
VALUES
  (
    13,
    App \ Models \ Cases,
    c7105402 - c5ad -4307 - bbfd - 36c6b5fc6a30,
    2,
    2023 -12 -29 09: 36: 02,
    2023 -12 -29 09: 36: 02
  )

**Reply by lucacastelnuovo** [reply]
@benbelgacemhichem did you manage to find a solution?

**Comment by DevYSM** [comment]
Any helps?
